### ISSUE TITLE
Missing APIs for Object Detection API

### ISSUE THREAD

**Issue by Oceania2018** [SciSharp/TensorFlow.NET#396]
- [x] [tf.assert_equal](https://www.tensorflow.org/api_docs/python/tf/debugging/assert_equal)
- [x] [tf.rank](https://www.tensorflow.org/api_docs/python/tf/rank)
- [x] [tf.sparse_tensor_to_dense](https://www.tensorflow.org/api_docs/python/tf/sparse/to_dense)
- [x] [tf.sparse_to_dense](https://www.tensorflow.org/api_docs/python/tf/sparse/SparseTensor)
- [x] [tf.gather](https://www.tensorflow.org/api_docs/python/tf/gather)
- [x] [tf.slice](https://www.tensorflow.org/api_docs/python/tf/slice)
- [x] [tf.group](https://www.tensorflow.org/api_docs/python/tf/group)
- [x] [tf.random_shuffle](https://www.tensorflow.org/api_docs/python/tf/random/shuffle)
- [x] [tf.get_variable_scope](https://www.tensorflow.org/versions/r1.14/api_docs/python/tf/get_variable_scope)
- [x] [tf.reduce_any](https://www.tensorflow.org/api_docs/python/tf/math/reduce_any)
- [x] [tf.init_scope](https://www.tensorflow.org/api_docs/python/tf/init_scope?hl=en)
- [x] [tf.truediv](https://www.tensorflow.org/api_docs/python/tf/math/truediv)
- [x] [tf.reduce_all](https://www.tensorflow.org/api_docs/python/tf/math/reduce_all)
- [x] [tf.control_dependencies](https://www.tensorflow.org/api_docs/python/tf/control_dependencies?hl=en)
- [x] [tf.check_numerics](https://www.tensorflow.org/api_docs/python/tf/debugging/check_numerics)
- [x] [tf.variance_scaling_initializer](https://www.tensorflow.org/api_docs/python/tf/initializers/variance_scaling)
- [x] [tf.zeros_like](https://www.tensorflow.org/api_docs/python/tf/zeros_like)
- [x] [tf.add_n](https://www.tensorflow.org/api_docs/python/tf/math/add_n)
- [x] [tf.PaddingFIFOQueue](https://www.tensorflow.org/api_docs/python/tf/queue/PaddingFIFOQueue?hl=en)
- [x] [tf.reverse_v2](https://www.tensorflow.org/api_docs/python/tf/reverse?hl=en)
- [x] [tf.batch_to_space_nd](https://www.tensorflow.org/api_docs/python/tf/batch_to_space_nd?hl=en)
- [ ] [tf.device](https://www.tensorflow.org/api_docs/python/tf/device?hl=en)
- [x] [tf.identity](https://www.tensorflow.org/api_docs/python/tf/identity?hl=en)
- [x] [tf.boolean_mask](https://www.tensorflow.org/api_docs/python/tf/boolean_mask)
- [x] [tf.dynamic_stitch](https://www.tensorflow.org/api_docs/python/tf/dynamic_stitch?hl=en)
- [x] [tf.reduce_prod](https://www.tensorflow.org/api_docs/python/tf/math/reduce_prod?hl=en)
- [x] [tf.add_to_collection](https://www.tensorflow.org/api_docs/python/tf/add_to_collections?hl=en)
- [ ] [tf.executing_eagerly](https://www.tensorflow.org/api_docs/python/tf/executing_eagerly?hl=en) **Won't support eager mode**.
- [ ] [tf.map_fn](https://www.tensorflow.org/api_docs/python/tf/map_fn)
- [x] [tf.floordiv](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/math/floordiv)
- [x] [tf.not_equal](https://www.tensorflow.org/api_docs/python/tf/math/not_equal)
- [x] [tf.ones_like](https://www.tensorflow.org/api_docs/python/tf/ones_like)
